src/main/java/dev/group4/repos/PotluckRepo.java

```diff
@@ -9,4 +9,6 @@
 @Repository
 public interface PotluckRepo extends JpaRepository<Potluck,String> {
     //TODO add custom method signatures
+
+    Potluck findPotluckByPotluck__id(String id);
```

Comment:
This needed a custom method? I thought there was an inbuilt one for the jpa to find by id

Reply:
does it? i dont really know much about that. happy to take it back out then

Reply:
I'll double check after the commit. I'm pretty sure it does but I'm not certain.